xla/tests/onednn_matmul_test.cc

```diff
@@ -599,6 +599,40 @@ TEST_F(MatmulTest, TestF32ConstantWeights) {
   )");
 }
 
+TEST_F(MatmulTest, SimpleTestBF16Gemv1) {
+  if (!IsSupportedType(PrimitiveType::BF16)) {
+    GTEST_SKIP() << "CPU does not support BF16.";
+  }
+
+  const char* matmul_module_str = R"(
+  HloModule matmul.test.bf16, entry_computation_layout={(bf16[1000,10000]{1,0},bf16[10000]{0})->bf16[1000]{0}}
```

Comment:
To make the test short and easier to read, let's remove the `entry_computation_layout` since it's not really needed. Same for other new test cases. (I'll change existing cases later.)
```suggestion
  HloModule matmul.test.bf16
```